File: content/0-basics/2-hello-world/aisdk/2-agent_anthropic.ts
import { generateText } from "ai";
import { anthropic } from "@ai-sdk/anthropic";

const LLM_MODEL = "claude-opus-4-6";

const { text } = await generateText({
  model: anthropic(LLM_MODEL),
  prompt: "What is the capital of France?",
});
console.log(text);
// "Paris."

File: content/0-basics/2-hello-world/langgraph/2-agent_anthropic.py
from langchain.agents import create_agent
from langchain_anthropic import ChatAnthropic

LLM_MODEL = "claude-opus-4-6"

model = ChatAnthropic(model=LLM_MODEL)

# create_agent builds a graph that loops: LLM → tools → LLM
# with no tools, it's a simple prompt → response
agent = create_agent(model, tools=[])

result = agent.invoke({
    "messages": [("user", "What is the capital of France?")]
})
print(result["messages"][-1].content)
# "Paris."
